index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <ul>
+                    <li><a class="myLogo font-style" href="">Kenbon</a> </li>
+                    <li><a class="toggler-icon"><img src="images/humburger.png" alt="humberger-icon"></a>
+                    </li>
+
+                </ul>
+            </nav>
+
+        </header>
+        <div class="main">
+            <h1 class="main_heading font-style">I'm Kenbon Glad to see you!</h1>
+            <p class="main-description font-style">
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
```

Comment:
- [ ] Good job so far 🥇, but I think it could be better if you wrap the title and description in a `<section>` tag, It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 🎲